MainDemo.Wpf/NavigationRail.xaml

```diff
@@ -0,0 +1,222 @@
+                    <TabControl Style="{DynamicResource MaterialDesignNavigatilRailTabControl}" materialDesign:ColorZoneAssist.Mode="PrimaryMid">
+                        <TabItem Style="{DynamicResource MaterialDesignNavigationRailTabItem}" >
+                            <TabItem.Header>
+                                <materialDesign:PackIcon Kind="Heart" Width="24" Height="24"/>
+                            </TabItem.Header>
+                            <TextBlock Padding="5" TextWrapping="Wrap" >
+                                <Run Text="tab 1 content. Default look and behaviors. "/>
+                                <LineBreak/>
+                                <LineBreak/>
+                                <Run FontStyle="Italic">
+                                    Neque porro quisquam est qui dolorem ipsum quia dolor sit amet, consectetur, adipisci velit...
+                                </Run>
+                            </TextBlock>
+                        </TabItem>
+                        <TabItem Style="{DynamicResource MaterialDesignNavigationRailTabItem}" >
+                            <TabItem.Header>
+                                <materialDesign:PackIcon Kind="Spade" Width="24" Height="24"/>
+                            </TabItem.Header>
+                            <TextBlock Padding="5" TextWrapping="Wrap" >
+                                <Run Text="tab 2 content. Default look and behaviors. "/>
+                                <LineBreak/>
+                                <LineBreak/>
+                                <Run FontStyle="Italic">
+                                    Praesent sed dui arcu. Vivamus porta auctor sagittis
+                                </Run>
+                            </TextBlock>
+                        </TabItem>
+                        <TabItem Style="{DynamicResource MaterialDesignNavigationRailTabItem}" >
+                            <TabItem.Header>
+                                <materialDesign:PackIcon Kind="Tree" Width="24" Height="24"/>
+                            </TabItem.Header>
+                            <TextBlock Padding="5" TextWrapping="Wrap" >
+                                <Run Text="tab 3 content. Default look and right aligned. "/>
+                                <LineBreak/>
+                                <LineBreak/>
+                                <Run FontStyle="Italic">
+                                    Praesent sed dui arcu. Vivamus porta auctor sagittis
+                                </Run>
+                            </TextBlock>
+                        </TabItem>
+                    </TabControl>
+                </smtx:XamlDisplay>
+
+            </Grid>
+        </GroupBox>
+        <GroupBox Grid.Column="1" Header="Custom Floating Content" Style="{DynamicResource MaterialDesignCardGroupBox}" materialDesign:ColorZoneAssist.Mode="Standard" Margin="5">
```

Comment:
I would change this header to better match the language in the material design spec.
```suggestion
        <GroupBox Grid.Column="1" Header="Floating Action Button" Style="{DynamicResource MaterialDesignCardGroupBox}" materialDesign:ColorZoneAssist.Mode="Standard" Margin="5">
```